docs/pages/tracing.rst

```diff
@@ -0,0 +1,63 @@
+Tracing
```

Comment:
This should be listed under `Integration`, let's rename it to "Development"

Reply:
`Tracing -> Development` or `Integration  -> Development`?

Reply:
Done!